x/liquidfarming/keeper/auction.go

```diff
@@ -30,13 +30,13 @@ func (k Keeper) PlaceBid(ctx sdk.Context, poolId uint64, bidder sdk.AccAddress,
 	}
 
 	if biddingCoin.Amount.LT(liquidFarm.MinBidAmount) {
-		return types.Bid{}, sdkerrors.Wrapf(types.ErrSmallerThanMinimumAmount, "%s is smaller than %s", biddingCoin.Amount, liquidFarm.MinBidAmount)
+		return types.Bid{}, sdkerrors.Wrapf(types.ErrNotBiggerThanMinimumAmount, "%s is not bigger than %s", biddingCoin.Amount, liquidFarm.MinBidAmount)
```

Comment:
Actually we should keep this line as-is, since the condition of the if statement is checking whether `biddingCoin.Amount` is actually **smaller** than `liquidFarm.MinBidAmount`.

Reply:
Thanks for the catch. I will revert it back.